bot/internal/bot/label.go

```diff
@@ -69,7 +80,14 @@ func (b *Bot) labels(ctx context.Context, files []github.PullRequestFile) ([]str
 	// The branch name is unsafe, but here we are simply adding a label.
 	if b.c.Environment.Repository != env.CloudRepo && isReleaseBranch(b.c.Environment.UnsafeBase) {
 		log.Println("Label: Found backport branch.")
-		labels = append(labels, "backport")
+		labels = append(labels, backportLabel)
+
+		// If automatic backport body doesn't have changelog entry it means there shouldn't be one, otherwise
+		// it would be automatically added. Mark PR with no-changelog label.
+		if len(b.getChangelogEntries(prBody)) == 0 && strings.HasPrefix(b.c.Environment.UnsafeHead, botBackportBranchPrefix) {
```

Comment:
As an alternative to this, should we just copy all labels (except the `backport` labels).  That way if `no-changelog` (and other attributes) are set on the original PR, they are defaulted to the backport.

Reply:
I've removed any changes to the label workflow in this PR, now setting the label on the backport workflow itself, as @r0mant suggested. I think we can actually do copying of all labels on the backport workflow, but I feel like that should be a separate PR if we decide to do that.